### PR TITLE
list-of-posts

### PR DESCRIPTION
**Pull request by VitalikStrog**
- [DEMO LINK](https://vitalikstrog.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by caelum29 — APPROVED** [review]
😻

File: src/components/Comments/Comments.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        name,
+        email,
+        id,
+        body,
+      }) => (
+        <div className={classNames(
+          'comment-post',
+          { 'comment-post--odd': id % 2 !== 0 },
+        )}
+        >
```
Comment (Anna-Tkachenko) [inline src/components/Comments/Comments.tsx:24]:
```suggestion
        <div key={...} className={classNames(
          'comment-post',
          { 'comment-post--odd': id % 2 !== 0 },
        )}
        >
```

File: src/components/PostList/PostList.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = (({ posts = [] }) => (
+  <ul className="posts-list">
+    {posts.map((post) => {
+      return post.user && (
+        <li className={`posts-list__item item ${post.id}`}>
```
Comment (Anna-Tkachenko) [inline src/components/PostList/PostList.tsx:16]:
```suggestion
        <li key={...} className={`posts-list__item item ${post.id}`}>
```

Comment (Anna-Tkachenko) [reply src/components/PostList/PostList.tsx:16]: return statement is redundant here

**Review by Anna-Tkachenko — APPROVED** [review]
🔥

**Pull request closed without merging**